equinix/resource_metal_virtual_circuit_acc_test.go

```diff
@@ -3,8 +3,8 @@ package equinix
 import (
 	"fmt"
 	"log"
-	"os"
 	"testing"
+	"os"
```

Comment:
this might not be gofmt'd - I expect "os" before "testing".

Perhaps this was applied from a patch before "testing" was added.